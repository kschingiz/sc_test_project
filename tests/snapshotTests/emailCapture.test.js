import React from "react";
import EmailCapture from "../../components/emailCapture";
import renderer from "react-test-renderer";

it("EmailCapture renders correctly", () => {
  const tree = renderer.create(<EmailCapture />).toJSON();
  expect(tree).toMatchSnapshot();
});
