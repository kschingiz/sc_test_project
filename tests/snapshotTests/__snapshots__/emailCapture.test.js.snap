// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailCapture renders correctly 1`] = `
<div
  className="makeStyles-signUpWrapper-1"
>
  <div
    className="makeStyles-captureTitle-2"
  >
    Do you want
    <br />
    <b>
      more attendees?
    </b>
  </div>
  <div
    className="makeStyles-captureSubtitle-3"
  >
    Join our mailing list for weekly tips on how to drive the right audience of professionals to your events.
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root makeStyles-emailInput-4"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
      data-shrink={false}
    >
      Email address
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
      />
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-65 MuiOutlinedInput-notchedOutline"
        style={
          Object {
            "paddingLeft": 8,
          }
        }
      >
        <legend
          className="PrivateNotchedOutline-legend-66"
          style={
            Object {
              "width": 0.01,
            }
          }
        >
          <span
            dangerouslySetInnerHTML={
              Object {
                "__html": "&#8203;",
              }
            }
          />
        </legend>
      </fieldset>
    </div>
  </div>
  <br />
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Sign up
    </span>
  </button>
</div>
`;
